src/main/java/com/google/api/generator/gapic/composer/utils/CommentFormatter.java

```diff
@@ -0,0 +1,62 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.api.generator.gapic.composer.utils;
+
+import com.google.api.generator.engine.ast.JavaDocComment;
+import com.google.common.base.Strings;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class CommentFormatter {
+
+  private CommentFormatter() {}
+
+  // Additional formatting logic for converting protobuf comment to Javadoc
+  public static String formatAndAddToJavaDocComment(String comment, String prefixPattern) {
```

Comment:
The method name may not reflect the latest changes, usually if you have an `and` in a method name, it's a code smell that either the method is doing too much or you can have a better name, maybe something like `fromProtobufCommentToJavaDocComment`?(I'm not good at naming as well, just throwing out an idea here)

Reply:
Thanks for catching this, I missed updating the method name when switching to returning a string instead of adding to a builder. Renamed to `formatAsJavaDocComment`.